src/composers/field.tsx

```diff
@@ -59,10 +62,11 @@ export function Field<Props>(WrappedComponent: IncomingField<Props>,
         }
 
         public render() {
+            const fieldState = this.props.form.getFieldState(this.getFieldName());
```

Comment:
Why are we storing a copy of the value if we're still going to call `getFieldState` in the render method?